hnn_core/utils.py

```diff
@@ -1,18 +0,0 @@
-"""Utils."""
-
-# Authors: Mainak Jas <mjas@harvard.mgh.edu>
-
-import platform
-import os.path as op
-
-from neuron import h
-
-
-def load_custom_mechanisms():
-    if platform.system() == 'Windows':
-        mech_fname = op.join(op.dirname(__file__), '..', 'mod', 'nrnmech.dll')
-    else:
-        mech_fname = op.join(op.dirname(__file__), '..', 'mod', 'x86_64',
-                             '.libs', 'libnrnmech.so')
-    h.nrn_load_dll(mech_fname)
-    print('Loading custom mechanism files from %s' % mech_fname)
```

Comment:
why did you delete this @blakecaldwell ?

Reply:
it's needed to run the scripts from outside the `hnn-core` directory and build the docs. I will check tomorrow if that still works

Reply:
Yes, it's in neuron.py, with all of the neuron stuff that depends on globals (:sigh).

With the MPI scheme, it's not necessary to call this until actually about to build the network.